target/linux/ramips/mt7621/base-files/sbin/fixup-mac-address

```diff
@@ -13,6 +13,10 @@ case $board in
 		partname=factory
 		offset=$((0xe000))
 	;;
+	hatlab,gateboard-one)
```

Comment:
The MAC address is stored in factory partition 0x0, but I don't know why it can't be written in OpenWrt.

```
root@OpenWrt:/# fixup-mac-address
Current MAC address: ca:ab:9d:f7:3c:53
Overwrite (y/N)? y
New MAC address: ce:b5:b3:fe:13:bb
Write to EEPROM (y/N)? y
[  506.568015] blk_update_request: I/O error, dev mtdblock2, sector 0 op 0x1:(WRITE) flags 0x800 phys_seg 1 prio class 0
[  506.578687] Buffer I/O error on dev mtdblock2, logical block 0, lost async page write
Done
```

And this board has 3 mini-PCIe slots, there is nothing installed on these slots by default.

Reply:
I try to use dd command to write factory partition also failed